invenio_communities/templates/semantic-ui/invenio_communities/details/members/members.html

```diff
@@ -18,10 +18,63 @@
 
 {%- block settings_body %}
 <div class="thirteen wide column">
-  <div
-    id="app"
-    data-community='{{ community | tojson}}'>
+  <div data-invenio-search-config='{
```

Comment:
Note for reviewers: This needs to be tuned.

Reply:
why the config is hardcoded in jinja?

Reply:
This is temporal config just to have the search view available, it will fixed in a next PR where we connect the backend with the view and inject the config using the helpers in `invenio-search-ui`

Reply:
:+1: